### ISSUE TITLE
bug: Bad Gateway after a few requests on fresh install

### ISSUE THREAD

**Issue by fxs38** [yogeshojha/rengine#1009]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

On a fresh install of rengine, I can login, and after a few actions on the UI I get a Bad Gateway error. The error seem to disappear after a restart of the containers, but I cannot start any scan due to this.
[rengine.log](https://github.com/yogeshojha/rengine/files/13062634/rengine.log)

### Expected Behavior

Normal browsing

### Steps To Reproduce

Browse to engine URL
Login
Click into a few menus
Bad Gateway error appears

### Environment

```markdown
- reNgine: 2.0-jasper
- OS: Ubuntu 23.04
- Python: 
- Docker Engine: 
- Docker Compose: 
- Browser: Chrome
```

**Comment by psyray** [comment]
Did the celery-beat container started ?

**Comment by psyray** [comment]
A lot of bugs have been fixed in 2.0.1, so update and check again, open a new issue if it persists

And check your specs :
- Good CPU (min i5)
- 8 threads
- 16Go RAM
- Fast SSD (nvme)
